lms/djangoapps/bulk_email/tasks.py

```diff
@@ -400,14 +402,17 @@ def _get_source_address(course_id, course_title):
     """
     course_title_no_quotes = re.sub(r'"', '', course_title)
 
-    # For the email address, get the course.  Then make sure that it can be used
-    # in an email address, by substituting a '_' anywhere a non-(ascii, period, or dash)
-    # character appears.
-    from_addr = u'"{0}" Course Staff <{1}-{2}>'.format(
-        course_title_no_quotes,
-        re.sub(r"[^\w.-]", '_', course_id.course),
-        settings.BULK_EMAIL_DEFAULT_FROM_EMAIL
+    # Translators: Bulk email from address e.g. ("Physics 101" Course Staff)
+    course_staff_title = _('"{course_name}" Course Staff')
+
+    from_addr_format = course_staff_title + u' <{course_id}-{from_email}>'
```

Comment:
I've made another test on Gmail by writing the following string in the `To` field:

The string in byte-order it looks like the following:

    ' '.join[
      'عمر',  # First name comes first
      'العيثاوي',  # Family name comes second
      '<oithawi@qrf.org>',  # Email comes last
    ] 

While editing (on focus):
![image](https://cloud.githubusercontent.com/assets/645156/12535404/cbeed770-c28a-11e5-9d8e-fe7333eabf21.png)


Finished editing (on blur):
![image](https://cloud.githubusercontent.com/assets/645156/12535405/cdeb2628-c28a-11e5-94a3-bd4a41bbe249.png)

As you can see the Gmail displays it correctly.

I'm sure this is the correct way and it would cause a bug to change the byte-order of the email and the name.